api/src/controllers/team.js

```diff
@@ -310,6 +315,114 @@ async function updateTeam(req, res, next) {
   }
 }
 
+/**
+  * Delete the team from the database on /team/:team_id
+  * @param {} req request object, containing team id in the url
+  * @param {*} res response object, the deleted team document
+  * @returns 200: teamn is deleted
+  * @returns 404: team is not found
+  * @returns 400: team id is not in a valid hexadecimal format
+  */
+async function deleteTeam(req, res, next) {
+  try {
+    const { team_id: _id } = req.params;
+
+    await HomePage.deleteOne({ teamId: _id });
+    await Website.deleteOne({ teamId: _id });
+    await Achievement.deleteOne({ teamId: _id });
+    await Publication.deleteOne({ teamId: _id });
```

Comment:
iirc this only deletes the first publication it finds, you'll need to use deleteMany()

Reply:
done